client/src/app/core/core-services/data-store.service.ts

```diff
@@ -323,15 +329,18 @@ export class DataStoreService {
      * Deletes all models that belong to a meeting.
      */
     public async clearMeetingModels(): Promise<void> {
+        const removedCollections = [];
         for (const collection of Object.keys(this.modelStore)) {
             if (this.modelMapper.isMeetingSpecificCollection(collection)) {
                 this.remove(
```

Comment:
I wonder why the repos do not get notified, the remove should work. I had a look: There is no updateslot running, when calling this from the active meeting service. Please have a look at `handleChangedAndDeletedModels` (`this.DSUpdateManager.getNewUpdateSlot` and `commit`) in the `autoupdate.service.ts`. Updateslots are the way that updates in DS are propagated to repos. `clear` is specially treated. You can decide: 
- Use an update slot and the event is not needed (not tested)
- Use the event, and do not call `remove`. The call can just be removed. Please call `triggerModifiedObservable` at the end of this method (as well as adding it to `clear`).

Reply:
I will take the second option. The reason is, that - when moving from a list to the dashboard - a user sees "missing content" in the list.